qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -469,9 +469,9 @@ status()
     for cluster in ${CLUSTER1} ${CLUSTER2}
     do
 	echo "${cluster} status"
-	ceph --cluster ${cluster} -s
-	ceph --cluster ${cluster} service dump
-	ceph --cluster ${cluster} service status
+	CEPH_ARGS='' ceph --cluster ${cluster} -s
+	CEPH_ARGS='' ceph --cluster ${cluster} service dump
+	CEPH_ARGS='' ceph --cluster ${cluster} service status
```

Comment:
Just wondering, taking how many `CEPH_ARGS` overrides we have now, couldn't it better to abandon the idea of initially setting `CEPH_ARGS` and explicitly use `--id ${CEPH_ID}` wherever it is needed?

Reply:
Out of the nearly 100 `ceph` or `rbd` CLI calls, only about 1/3 have the `CEPH_ARGS` override (albeit nearly all the `ceph` ones have it).

Reply:
Ok, good point about using a dedicated 'client.mirror' user for rbd commands by default instead of 'client.admin' to catch permission issues. But isn't it a bug in `create_users` test helper then that we are creating 'client.mirror' user without `mgr 'rbd profile'` caps? Yes it helped in this particular case to catch the bug in `rbd` (though "bug" may be too strong word for it, because it would not have failed for a user configured according to our documentation), but in general I think it is wrong that we are testing `rbd` using a not properly configured user.

Reply:
Agreed -- we should be using our nice cap profiles. However, that wouldn't have stopped the issue since our profiles don't give the average user access to see Ceph cluster services:

> $ ceph auth get-or-create client.mirror 'mon' 'profile rbd' 'osd' 'profile rbd' 'mgr' 'profile rbd' >> keyring
> $ rbd --id mirror mirror pool status --verbose
> rbd: failed to query services: (13) Permission denied

Reply:
Added the 'mgr' profile caps just to be consistent w/ the docs